cypress-tests/cypress/e2e/PaymentUtils/Nexixpay.js

```diff
@@ -184,5 +296,332 @@ export const connectorDetails = {
         },
       },
     },
+    MandateMultiUse3DSAutoCapture: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulThreeDSTestCardDetails,
+        },
+        currency: "EUR",
+        mandate_data: multiUseMandateData,
+        billing: billingAddress,
+      },
+      Response: {
+        status: 200,
+        trigger_skip: true,
```

Comment:
You will have to change the implementation once https://github.com/juspay/hyperswitch/pull/6588 gets merged here and else where for `trigger_skip`